packages/core/src/features/gdcapi/gdcgraphql.ts

```diff
@@ -84,7 +90,11 @@ export const graphqlAPISlice = coreCreateApi({
     try {
       results = await graphqlAPI(request.graphQLQuery, request.graphQLFilters);
     } catch (e) {
-      return { error: e as GraphQLFetchError };
+      return {
+        error: errorIsGraphQLError(e as Error | GraphQLFetchError)
```

Comment:
This is to fix an issue where network failures were adding unserializable objects to the store.